### PR TITLE
add new board HAKRCF405V2

### PR DESCRIPTION
**Pull request by hakrc1**
add a new board HAKRCF405V2

### CODE REVIEW

File: Tools/AP_Bootloader/board_types.txt [hunk 1]
```diff
@@ -267,6 +267,7 @@ AP_HW_ORAQF405PRO                    1155
 AP_HW_CBU_StampH743                  1156
 AP_HW_FOXEERF405_V2                  1157
 AP_HW_CSKY405                        1158
+AP_HW_HAKRCF405V2                    1159
```
Comment (peterbarker) [inline Tools/AP_Bootloader/board_types.txt:270]: Might be a good idea to create a separate PR which nabs this ID, just in case this PR takes a while to get in.

Comment (hakrc1) [reply Tools/AP_Bootloader/board_types.txt:270]: Thank you for your help.

**Comment by hakrc1** [comment]
@peterbarker  Hi,how to get official support for a new board？

**Comment by andyp1per** [comment]
> Hi,how to get official support for a new board？

@hakrc1 I can help you here. The best way to do this is become an ardupilot partner (https://ardupilot.org/ardupilot/docs/common-partners-program.html). If you want this done quickly then I can help you as part of my consulting business (www.fossuav.com) and if you are happy to get this done as and when then send me the hardware and I will take a look when I have time.
